modules/internal/ChapelArray.chpl

```diff
@@ -495,6 +500,7 @@ module ChapelArray {
   }
 
 
+  pragma "no copy return"
```

Comment:
This "no copy return" seems to be not so obvious to me. I'm often confused by runtime types but if you know something about why it's necessary here I think a comment is called for.